client/dashboard/me/billing-history/utils.tsx

```diff
@@ -62,6 +62,22 @@ export const groupDomainProducts = ( originalItems: ReceiptItem[] ) => {
 	return [ ...otherProducts, ...groupedDomainProducts ];
 };
 
+interface summarizedReceiptItems {
```

Comment:
By convention types are capitalized.

```suggestion
interface SummarizedReceiptItems {
```